### DISCUSSION TITLE
Persistent cache for webpack ---watch, after initial build use memory

### DISCUSSION THREAD

**Discussion by matzeeable** [webpack/webpack#12250] [Q&A] [answered]
Hey community!

If I start `webpack --watch` locally, it takes a long time for the initial build (the rebuild is very fast 😊).

Is it somehow possible, to use the persistent disk cache for the initial build, and the memory for the rebuilds?

Regards
Matthew

**Comment by alexander-akait** [comment, marked as answer]
Sounds good for improvements, I think this behavior can be default, can you open an issue?

**Reply by matzeeable** [reply]
Thanks for your reply @alexander-akait ! Created #12251.
